dvc/remote/base.py

```diff
@@ -413,7 +419,7 @@ def upload(self, from_info, to_info, name=None, no_progress_bar=False):
 
         name = name or from_info.name
 
-        self._upload(
+        self._upload(  # noqa, pylint: disable=no-member
```

Comment:
I think, we should create an `_upload`/`_download` function in the base class that just throws `RemoteActionNotImplemented` error. It would make this cleaner (without ignore, that is), and provide stronger checks-and-balances.